### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by qf3l3k**
commit hash fixed in binaries download link to point to testnet 15 version.

### CODE REVIEW

**Comment by loredanacirstea** [comment]
@qf3l3k , thank you.

I am curious - did it download the wrong version of the binary for you?
In practice, I've found that adding the commit does not have an effect (it will download the latest version anyway), except when the system has a cache. So, adding the commit forces a new request, hence it is good to have the right commit.

File: mythos-testnet/README.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ rm -rf /root/mythos
 ```
 
 ```shell=
-mkdir mythos && cd mythos && wget "https://github.com/loredanacirstea/tempreleases/raw/main/mythos-testnet/linux_x86_64.zip?commit=b0cc7196927a57d31cc8fc6fc864fc8f3c85d920" -O linux_x86_64.zip && unzip linux_x86_64.zip && mv linux_x86_64 ./bin && cd bin && chmod +x ./mythosd && cd ..
+mkdir mythos && cd mythos && wget "https://github.com/loredanacirstea/tempreleases/raw/main/mythos-testnet/linux_x86_64.zip?commit=0e4109955ec06886a058765f6cf2a260d1218b59" -O linux_x86_64.zip && unzip linux_x86_64.zip && mv linux_x86_64 ./bin && cd bin && chmod +x ./mythosd && cd ..
```
Comment (loredanacirstea) [inline mythos-testnet/README.md:44]: Actually, this commit should be the commit from this repo, with the latest version of the binary (fixed in https://github.com/loredanacirstea/tempreleases/commit/94052aae3cb44a8cdb74350acda0b3acfa29c59b).

**Comment by qf3l3k** [comment]
When using download with commit: b0cc7196927a57d31cc8fc6fc864fc8f3c85d920 I got AppHash error at height 10642.
When sync'd chain with 0e4109955ec06886a058765f6cf2a260d1218b59 all works fine.
Tried about 5 times and always got AppHash error at same height,
That refers to mythos_7000-15.

**Comment by loredanacirstea** [comment]
@qf3l3k , ok. So, due to caching, the older binary was downloaded. Make sure to run the version --long command and compare the commit & the genesis checksum before syncing, to catch this type of issue early.

**Pull request merged**